src/page-modules/departures/details/details.module.css

```diff
@@ -0,0 +1,100 @@
+.container {
+  margin: 0 auto;
+  max-width: var(--maxPageWidth);
+  padding: var(--spacings-xLarge);
+
+  display: grid;
+  grid-template-columns: 2fr 3fr;
+  grid-template-areas:
+    'header header'
+    'serviceJourney map';
+  gap: var(--spacings-small);
+}
+@media (max-width: 650px) {
+  .container {
+    grid-template-columns: 1fr;
+    grid-template-areas:
+      'header'
+      'serviceJourney'
+      'map';
+  }
+}
+.headerContainer {
+  grid-area: header;
+  display: flex;
+  flex-direction: column;
+  gap: var(--spacings-large);
+}
+.mapContainer {
+  grid-area: map;
+  max-height: 37.5rem;
+  width: 100%;
+  margin: 0 auto;
+}
+.serviceJourneyContainer {
+  grid-area: serviceJourney;
+}
+@media (max-width: 650px) {
+  .serviceJourneyContainer {
+    margin: 0 auto;
+  }
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  gap: var(--spacings-large);
+}
+
+.realtimeText {
+  display: flex;
+  align-items: center;
+  gap: var(--spacings-small);
+  font-size: 0.875rem;
+  color: var(--text-colors-secondary);
+}
+
+.callRows {
+  margin-top: var(--spacings-large);
+}
+.callRows__container {
+  padding: var(--spacings-medium);
+}
+.rowContainer {
+  position: relative;
+  --labelWidth: 5rem;
+  --decorationContainerWidth: 1.25rem;
+  --decorationLineWidth: 0.25rem;
+  --decorationLineEndWidth: 0.75rem;
+}
+.boardingInfo {
+  color: var(--text-colors-secondary);
+}
+.collapseButton {
+  padding-bottom: var(--spacings-medium);
```

Comment:
This padding makes the focus outline a bit weirder than it already is (IMO this button mode should have _some_ padding to make the outline be a tad bit better).

<img width="201" alt="2023-11-10 at 14 28 35@2x" src="https://github.com/AtB-AS/planner-web/assets/14045515/2fffc7f1-1fc6-43ec-870f-0d82dfa47dc1">

Reply:
Fixed 🎉